auditLogMover/package.json

```diff
@@ -20,7 +20,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "jest": "^26.6.3",
     "serverless": "^1.73.1",
-    "serverless-plugin-typescript": "^1.1.9",
```

Comment:
Serverless with compile our TS to JS now without this plugin?

Reply:
Yep, `serverless-bundle` now compiles `TS` for us
https://www.npmjs.com/package/serverless-bundle